lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -336,11 +342,19 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
     Livebook.Hubs.create_secret(secret)
   end
 
+  defp grant_access(secrets, secret_name, :hub, socket) do
+    secret = Enum.find(secrets, &(&1.name == secret_name and match?({:hub, _}, &1.origin)))
+
+    if secret,
+      do: Livebook.Session.set_secret(socket.assigns.session.pid, secret),
+      else: :ok
```

Comment:
Session.set_secret always returns :ok so there's less of a need for :else and because we never check the return value of this function, there's in fact no need for :else :)